### ISSUE TITLE
Flash in Header

### ISSUE THREAD

**Issue by triengel** [TwoBeers/shiword#27]
How do I put a flash file in place of the image file in the header? Currently there is no way to do this.

**Comment by jimo** [comment]
Open header.php and insert the flash code inside the "headerimg" div

**Comment by triengel** [comment]
Thank you! I was able to install the flash now I have a new issue maybe you can help me with. Every page I click on the flash is re booted and starts over again. I would just like it to play only once when someone first goes to the site. How do I configure this? Also, when I scroll the page up the flash header goes in front of the boarder instead of underneath like it should. Here is an example of the script I am using: 
![script2](https://f.cloud.github.com/assets/4370106/481574/2a15f232-b868-11e2-9aef-dff59cd60800.jpg)

Thanx, Jeff

**Comment by jimo** [comment]
Check your code, the `<div="` before the swf code means nothing.

> Every page I click on the flash is re booted and starts over again. I would just like it to play only once when someone first goes to the site.

this can be done using php sessions( http://w3schools.com/php/php_sessions.asp ) or cookies( http://w3schools.com/php/php_cookies.asp ), but here I can't help you.

> when I scroll the page up the flash header goes in front of the boarder instead of underneath like it should.

If you give me your url I can take a live check

**Comment by jimo** [comment]
no reply -> closed
